pkgs/top-level/python-packages.nix

```diff
@@ -13833,23 +13833,28 @@ in modules // {
 
   plover = buildPythonPackage rec {
     name = "plover-${version}";
-    version = "3.0.0";
+    version = "3.0.0+199";
     disabled = !isPy27;
```

Comment:
if it is a release, then please us the full version of it. If it is a snapshot, then write `plover-unstable-<date>` following the updated guidelines.